code/game/objects/items/flame/_flame.dm

```diff
@@ -115,7 +115,7 @@
 
 	return ..()
 
-/obj/item/flame/proc/extinguish(var/mob/user, var/no_message)
+/obj/item/flame/extinguish_fire(mob/user, no_message = FALSE)
```

Comment:
not sure how to feel about this. i feel like a lighter should be able to catch fire separately from it being on/off or lit/unlit? idk